etc/inc/disable-programs.inc

```diff
@@ -714,6 +717,7 @@ blacklist ${HOME}/.icedove
 blacklist ${HOME}/.imagej
 blacklist ${HOME}/.inkscape
 blacklist ${HOME}/.itch
+blacklist ${HOME}/.ivy2
```

Comment:
This is used by Apache Ivy (and maybe also maven and gradle) and so could
potentially impact any Java development tool (including IDEs).